sammba/data_fetchers/atlas.py

```diff
@@ -206,9 +206,6 @@ def fetch_masks_dorr_2008(image_format='nifti', downsample='30',
 
     corpus_callosum_labels = labels[
```

Comment:
see comment below but this would all be a lot clearer and more robust with pandas